lib/test-runner/mocha/separate/acceptance.js

```diff
@@ -25,10 +25,17 @@ function testFeature(feature) {
                   this.skip();
                 } else {
                   let self = this;
-                  let promise = new Ember.RSVP.Promise(function(resolve) { yadda.Yadda(library.default(), self).yadda(step, context, resolve); });
-                  promise.catch(function() {
-                    failed = true;
+                  let promise = new Ember.RSVP.Promise(function(resolve) {
+                    yadda.Yadda(library.default(), self).yadda(step, context, function next(err, result) {
+                      if (err) {
+                        failed = true;
+                        throw err;
+                      }
+
+                      return result;
```

Comment:
This should be `resolve(result);` will raise a PR

Reply:
🤦‍♂️